docs/get-started/index.md

```diff
@@ -0,0 +1,313 @@
+    <h3>Angular</h3>
+  </a>
+  <a href="./sveltekit/?renderer=svelte" class="framework card">
+    <img src="./logo-svelte.svg" width="28" height="28" alt="" class="img" />
+    <h3>SvelteKit</h3>
+  </a>
+  <a href="./svelte-vite/?renderer=svelte" class="framework card">
+    <img src="./logo-svelte.svg" width="28" height="28" alt="" class="img" />
+    <div>
+      <h3>Svelte</h3>
+      <p>with Vite</p>
+    </div>
+  </a>
+  <a href="./svelte-webpack5/?renderer=svelte" class="framework card">
+    <img src="./logo-svelte.svg" width="28" height="28" alt="" class="img" />
+    <div>
+      <h3>Svelte</h3>
+      <p>with Webpack</p>
+    </div>
+  </a>
+  <a href="./web-components-vite/?renderer=web-components" class="framework card">
+    <img src="./logo-web-components.svg" width="28" height="28" alt="" class="img" />
+    <div>
+      <h3>Web Components</h3>
+      <p>with Vite</p>
+    </div>
+  </a>
+  <a href="./web-components-webpack5/?renderer=web-components" class="framework card">
+    <img src="./logo-web-components.svg" width="28" height="28" alt="" class="img" />
+    <div>
+      <h3>Web Components</h3>
+      <p>with Webpack</p>
+    </div>
+  </a>
+</div>
+
+Want to know more about the installation process? Check out the [installation guide](./install.md).
+
+## Main concepts
+
+Storybook is a powerful tool that can help you with many aspects of your UI development workflow. Here are some of the main concepts to get you started.
+
+<div class="sb-grid two-up">
+  <a href="../writing-stories/index.md" class="card card-large concept">
+    <img src="./icon-story.svg" alt="" width="20" height="20" class="img" />
+    <h3>Stories</h3>
+    <p>A story captures the rendered state of a UI component. Each component can have multiple stories which describe all the "interesting" states that component supports.</p>
+  </a>
+  <a href="../writing-docs/index.md" class="card card-large concept">
+    <img src="./icon-docs.svg" alt="" width="20" height="20" class="img" />
+    <h3>Docs</h3>
+    <p>Storybook automatically creates documentation for your components using your stories. That allows you to create UI library usage guidelines, design system sites, and more.</p>
+  </a>
+  <a href="../writing-tests/index.md" class="card card-large concept">
+    <img src="./icon-testing.svg" alt="" width="20" height="20" class="img" />
+    <h3>Testing</h3>
+    <p>Stories are a pragmatic starting point for your UI testing strategy. You already write stories as a natural part of UI development, testing those stories is a low-effort way to prevent UI bugs over time.</p>
+  </a>
+  <a href="../sharing/index.md" class="card card-large concept">
+    <img src="./icon-sharing.svg" alt="" width="20" height="20" class="img" />
+    <h3>Sharing</h3>
+    <p>Publishing your Storybook allows you to share your work with others. It can also be embedded in other places like Notion or Figma.</p>
```

Comment:
Change second sentence to: ‘You can also embed your stories in places like Notion and Figma.’